### PR TITLE
Switch docker image to new 8.x-php8.4 tag.

### PR DESCRIPTION
**Pull request by kporras07**
I think we should be using a Pantheon official image and ideally, that image should contain terminus and build tools in it. I'm switching the image in this PR.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -41,23 +41,10 @@ jobs:
       - name: Code sniff
         run: composer run-script code:lint
 
-      - name: Install Terminus
```
Comment (jazzsequence) [inline .github/workflows/ci.yml:44]: Is terminus already installed at another step?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -41,23 +41,10 @@ jobs:
...
-        run: |
-          wget https://github.com/pantheon-systems/terminus/releases/download/3.0.0-beta1/terminus
-          chmod +x terminus
-          mv terminus /usr/local/bin/t3
-          mkdir ~/.terminus
-          t3 self:info
-
-      - name: Install Terminus Build Tools
-        run: |
-          t3 self:plugin:install terminus-build-tools-plugin
```
Comment (jazzsequence) [inline .github/workflows/ci.yml:54]: These are used so in some of the Robo workflows. Is there some other place that the build tools are being installed?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kporras07) [reply .github/workflows/ci.yml:44]: It's in the docker image

File: .github/workflows/ci.yml (hunk 2 above)
Comment (kporras07) [reply .github/workflows/ci.yml:54]: It's in the docker image

**Review by jazzsequence — APPROVED** [review]
If these things are already in the image then this looks g2g to me. 👍

**Pull request merged**